LeadKit/Sources/Extensions/Array/Array+Extensions.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Copyright (c) 2017 Touch Instinct
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+public extension Array where Element: Equatable {
+
+    /// Union array with another arrays, without element duplication
+    func union(values: [Array.Element]...) -> Array {
+        var result = self
+
+        for array in values {
```

Comment:
Only when it dramatically reduce amount of code.
for example:

```swift

let items = ["1", "2", "3"]
let labels = [lbl1, lbl2, lbl3]

for (item, label) in zip(items, label) {
    label.text = item
}
```


```swift
zip([lbl1, lbl2, lbl3], ["1", "2", "3"]).forEach { $0.0.text = $0.1 }
```

But `for` loop is more performance efficient. For unknown array size it's better to use `for` loop